Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -869,38 +878,37 @@ ioi ld   (NBCR),a
    inc   hl
    ld    (ix+[_decb]+pd_rx_error),hl
 
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Again, where in the R6000 is this documented?
+#if !defined(_RAB6K)

```

Comment:
I think you should revert this section.  Bit 5 of `rcvlsbstat` is "length error" and bit 4 is "CRC Error".